clients/client-accessanalyzer/typedoc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "../../typedoc.client.json"
+  "extends": ["../../typedoc.client.json"],
+  "entryPoints": ["src/index.ts"],
+  "out": "docs",
+  "readme": "README.md"
```

Comment:
The `out` and `readme` configuration may be used from root typedoc.json
We might take up this work as a follow-up.